### DISCUSSION TITLE
Suggestion: Popups

### DISCUSSION THREAD

**Discussion by MarlonGrazek** [discordjs/discord.js#5963] [Ideas]
It would be nice if you would be able to create popups such as the one you get when you want to ban someone from a guild.
You can create those popups, give it a title and description and add images, buttons, textboxes and selectors.
It would make commands much easier to handle and also it would look much more clean than messages.

Example: 
![Screenshot 2021-06-29 162813](https://user-images.githubusercontent.com/80133497/123815738-05289700-d8f7-11eb-9777-fdc4085789ad.png)

**Comment by olivermrose** [comment]
DJS has no control over this. Suggest features in the [DAPI docs repo](https://github.com/discord/discord-api-docs).

**Reply by GoldenAngel2** [reply]
https://feedback.discord.com not the API Docs repo. Since it's a client feature being suggested.
